test/app.router.js

```diff
@@ -1130,6 +1130,54 @@ describe('app.router', function(){
     var app = express();
     assert.strictEqual(app.get('/', function () {}), app)
   })
+
+  it('should should not use disposed router/middleware', function(done){
```

Comment:
could we just link to the conversation where this was asked about so we know why this behavior deserves a test?

Reply:
added